winter/pagination/utils.py

```diff
@@ -0,0 +1,44 @@
+from typing import Optional
+
+from rest_framework.request import Request as DRFRequest
```

Comment:
why alias? to distinguish from Django Request? why not namespace?

Reply:
Yes, for distinguish from Django Request
if I use namespace I should use alias for it too

Reply:
I mean:
import rest_framework.request
and then:
rest_framework.request.Request

Reply:
I think it is too long